rafx-assets/src/assets/graphics_pipeline/assets.rs

```diff
@@ -247,7 +247,7 @@ impl MaterialPassData {
                 return Err(error)?;
             }
 
-            log::trace!("  Reflection data:\n{:#?}", reflection_data);
+            //log::trace!("  Reflection data:\n{:#?}", reflection_data);
```

Comment:
Commented out?

Reply:
yeah these are super spammy, even for trace